CodePush.m

```diff
@@ -111,6 +119,20 @@ + (BOOL)isUsingTestConfiguration
     return testConfigurationFlag;
 }
 
+/*
+ * This returns the modified date as a string for a given file URL.
+ */
++ (NSString *)modifiedDateStringOfFileAtUrl:(NSURL *)fileUrl
```

Comment:
This should be `*URL` (including the `fileURL` parameter name. Additionally, this should be in the private API section right?